datafusion/core/tests/sql/select.rs

```diff
@@ -639,7 +639,7 @@ async fn query_nested_get_indexed_field_on_struct() -> Result<()> {
     ctx.register_table("structs", table_a)?;
 
     // Original column is micros, convert to millis and check timestamp
-    let sql = "SELECT some_struct[\"bar\"] as l0 FROM structs LIMIT 3";
+    let sql = "SELECT some_struct['bar'] as l0 FROM structs LIMIT 3";
```

Comment:
See https://github.com/apache/arrow-datafusion/pull/2500#discussion_r869250295 for an explanation of this change